src/screens/SignIn/index.tsx

```diff
@@ -12,13 +16,13 @@ const SignIn = (): JSX.Element => {
   }
 
   return (
-    <div>
-      <img alt="NIMBLE" />
-      <p>Sign in to Nimble</p>
-
-      {error && <div>{error}</div>}
-      <SignInForm submitHandler={handleSubmit} />
-    </div>
+    <Background backgroundImage={authBackground}>
+      <div className="container-sign-in">
+        <AuthLogo label="Sign in to Nimble" />
+        {error && <Alert title="Unable to sign in" detail={error} />}
```

Comment:
Do we need to get the title from props? As it seems that `Unable to sign in` will be shown always 🤔

Reply:
I'm not sure I totally understand you. I think we don't need to get the error title from props because what you mentioned that `Unable to sign in` is always show in the case of sign in unsuccessfully so it is static, not dynamic.